tests/task/task_runner/test_standard_task_runner.py

```diff
@@ -228,6 +225,8 @@ def test_on_kill(self):
         for process in processes:
             assert not psutil.pid_exists(process.pid), f"{process} is still alive"
 
+        session.close()
```

Comment:
Should it be in try/finally clause? Or maybe even setup/tearDown?

Reply:
I can convert this to `createSession` which would do that, but calling `close` like this is pretty common in our test suite.

Reply:
I see a number of similar uses (but a number of finally ones as well). Actually not having finally might be a reason why we are having side effects/flaky tests so we might want to fix them all.

Reply:
Actually it does call for a `with newSession() as session` ... Should we make ContextManager for that? That would avoid try/finally.

Reply:
Maybe simply:

```
from contextlib import closing

with closing(new Session) as session:

Reply:
I like how your code blocks are all in a Java-ish language 😆

Reply:
some things are never forgotten :D.

Reply:
I also still remember this: `ADD X TO Y GIVING Z.` (obligatory . at the end). Guess what language it is.

Reply:
That was true `literate programming` :D. which is becoming fashionable again.

Reply:
Fixup commit added that uses `create_session` 👍